es6/ops/random.js

```diff
@@ -68,14 +62,14 @@ class RandomFloat extends PlanOutOpRandom {
 }
 
 class RandomInteger extends PlanOutOpRandom {
+  compatRandomIntegerCalculation(minVal, maxVal) {
```

Comment:
I wonder if the PlanOutOpRandom base class should just have a single compatOp function that each class overrides instead of having each of these individually named functions for each class.